allennlp_models/rc/common/reader_utils.py

```diff
@@ -69,7 +69,7 @@ def char_span_to_token_span(
     token_span : ``Tuple[int, int]``
         `Inclusive` span start and end token indices that match as closely as possible to the input
         character spans.
-    error : ``bool``
+    exact : ``bool``
```

Comment:
Yeah, probably.

Reply:
No, wait. That flips the meaning around. We return `False` if everything is fine, `True` otherwise. I don't want to change it. It breaks backwards compatibility for a minuscule gain.

Reply:
Then I think the parameter description is wrong

Reply:
You are right! I put in a fix.

Reply:
What about the new test?